astropy/table/table.py

```diff
@@ -2021,6 +2021,70 @@ def group_by(self, keys):
 
         return groups.table_group_by(self, keys)
 
+    def to_pandas(self):
+        """
+        Return a :class:`pandas.DataFrame` instance
+
+        Returns
+        -------
+        dataframe : :class:`pandas.DataFrame`
+            A pandas :class:`pandas.DataFrame` instance
+        """
+        from pandas import DataFrame
```

Comment:
I think here it is reasonable not to try/except -- too much clutter. However, one could add a `Raises` section in the docstring, which includes the `ImportError:` as well as the two `ValueError`.  Indeed, the documentation should warn one way or another that multi-d and mixin columns are not (yet) supported, and that masked columns are converted.